cirq/circuits/circuit.py

```diff
@@ -1559,6 +1559,11 @@ def save_qasm(self,
         """
         self._to_qasm_output(header, precision, qubit_order).save(file_path)
 
+    @classmethod
+    def from_qasm(cls, qasm: str):
+        from cirq.contrib import QasmCircuitParser
```

Comment:
Don't reference contrib from inside non-contrib.

Reply:
ah, alright, I can totally remove this. 
I figured it would be a useful utility method.